python/ray/serve/api.py

```diff
@@ -890,6 +894,57 @@ async def __del__(self):
     return decorator
 
 
+@PublicAPI(stability="alpha")
+class DAGHandle:
+    """Resolved from a DeploymentMethodNode at runtime.
+
+    This can be used to call the DAG from a driver deployment to efficiently
+    orchestrate a multi-deployment pipeline.
+    """
+
+    def __init__(self, serialized_dag_json: str):
+        raise NotImplementedError()
+
+    def remote(self, *args, **kwargs) -> ray.ObjectRef:
+        """Call the DAG."""
+        raise NotImplementedError()
+
+
+@PublicAPI(stability="alpha")
+class DeploymentMethodNode(ClassNode):
```

Comment:
subclass `DAGNode` is better here, even current `ClassMethodNode` don't use much of `ClassNode` implementation

later we likely also convert plain task / function to `DeploymentNode` that wraps a func_or_class, so better not to assume it's always an instance of ClassNode.

Reply:
done